pkg/storage/file/keyspaces.go

```diff
@@ -26,8 +26,119 @@ const (
 	prefixRecordKeySpace
 	prefixRecordIndexByTypeVersionKeySpace
 	prefixRegistryServiceKeySpace
+	prefixIndexableFieldsKeySpace
 )
 
+// indexable fields key space
+//
+//	keys: prefix-reference-graph | 0x00 |{record-type as bytes} | {from-field as bytes} | 0x00 | {from-value as bytes} | 0x00 | {record_id as bytes}
```

Comment:
The prefix is a single byte so there's no `0x00` after it. But there should be one after the record type.

```suggestion
//	keys: prefix-reference-graph | {record-type as bytes}| 0x00 | {from-field as bytes} | 0x00 | {from-value as bytes} | 0x00 | {record_id as bytes}
```